File: algorithms/0088.merge-sorted-array/solution.rs
impl Solution {
    pub fn merge(nums1: &mut Vec<i32>, m: i32, nums2: &mut Vec<i32>, n: i32) {
        nums1.drain((m as usize)..(nums1.len() as usize));
        for num in nums2 {
            nums1.push(*num);
        }
        nums1.sort()
    }
}

File: algorithms/0088.merge-sorted-array/solution.rb
# @param {Integer[]} nums1
# @param {Integer} m
# @param {Integer[]} nums2
# @param {Integer} n
# @return {Void} Do not return anything, modify nums1 in-place instead.
def merge(nums1, m, nums2, n)
  nums1.slice!(m, nums1.size)
  nums2.each { |num| nums1 << num }
  nums1.sort!
end
